namer/namer.cc

```diff
@@ -1477,23 +1477,14 @@ class SymbolDefiner {
         }
     }
 
-public:
-    SymbolDefiner(State state, const core::FoundDefinitions &foundDefs, optional<core::FoundDefHashes> oldFoundHashes)
-        : state(move(state)), foundDefs(foundDefs), oldFoundHashes(move(oldFoundHashes)) {}
-
-    SymbolDefiner::State run(core::MutableContext ctx) {
+    void beginIncrementalDefinitionInternal(core::MutableContext ctx) {
```

Comment:
Slight preference to give some more descriptive names for these functions, to give a mental model to future developers whether to put new logic in `begin` or in `finish`.

Reply:
I split things up into three parts so that the "begin" phase didn't get a gigantic name.